src/db/migrations/20190408115030-add-validation-columns-users.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        await true
+        await queryInterface.addColumn('Users', 'contactMethodVerificationToken', {
+            type: Sequelize.STRING,
+            unique: true,
+        });
+        await queryInterface.addColumn('Users', 'contactMethodVerificationTokenExpires', {
+            type: Sequelize.DATE,
+        });
+        await queryInterface.addColumn('Users', 'contactMethodToVerify', {
+            type: Sequelize.STRING, // eslint-disable-line new-cap
+            defaultValue: '',
+        });
+        await queryInterface.addColumn('Users', 'verifiedContactMethods', {
+            type: Sequelize.ARRAY(Sequelize.STRING), // eslint-disable-line new-cap
+            defaultValue: [],
+        });
+        await queryInterface.sequelize.query(`UPDATE "Users" SET "verifiedContactMethods" = ARRAY["email"] WHERE "Users"."username" = "Users"."email"`)
```

Comment:
Sometimes there will be users in the DB whose username is not the same as their email address. The `WHERE` clause here will prevent this script from working for those users (I tested and confirmed this). (Also, I don't see the point of the `WHERE` clause; what is it for?) IMO the `WHERE` clause should be removed.

Reply:
Fixed. Tested. Working. Resolving.